sensors/triggers/registry.go

```diff
@@ -0,0 +1,24 @@
+package triggers
+
+import (
+	"fmt"
+
+	"github.com/argoproj/argo-events/common"
+	apicommon "github.com/argoproj/argo-events/pkg/apis/common"
+	"github.com/riferrei/srclient"
+)
+
+// GetSchemaFromRegistry returns a schema from registry.
+func GetSchemaFromRegistry(sr *apicommon.SchemaRegistryConfig) (*srclient.Schema, error) {
```

Comment:
We don't need this file, the function is particularly for kafka trigger, putting it in the kafka trigger is better.